app/scripts/components/exploration/components/datasets/color-range-slider.scss

```diff
@@ -0,0 +1,62 @@
+
+.slider__track {
+  background-color: #dddfe2;
+}
+
+.slider__range {
+  background-color: #1565ef;
```

Comment:
From a recent discussion with @faustoperez I think we would be fine if we use USWDS color utilities that approximate these hardcoded hexes, even if our designs don't strictly follow the USWDS color tokens yet (at least features that were designed a while ago).

I had a similar situation with the `react-calendar`, where the Figma designs used custom hex values, but we wanted to lean towards and try using their approximate USWDS counterparts. For this particular hex here (`#1565ef`) I suggest we use the `primary-vivid` token which comes pretty close (e.g. usage [here](https://github.com/NASA-IMPACT/veda-ui/blob/46e1c5df246aafec25b760e231938607cd1cd41e/app/scripts/components/common/datepicker/index.scss#L27). Sigh, I opened an issue to optimize the paths of these imports as well: https://github.com/NASA-IMPACT/veda-ui/issues/1196).


List of USWDS colors and tokens: https://designsystem.digital.gov/utilities/color/